packages/cms-cli/commands/customObject/create.js

```diff
@@ -12,11 +12,11 @@ const { setLogLevel, getPortalId } = require('../../lib/commonOpts');
 const { logDebugInfo } = require('../../lib/debugInfo');
 const { batchCreateObjects } = require('@hubspot/cms-lib/api/customObject');
 
-exports.command = 'create <objectTypeId> <definition>';
+exports.command = 'create <name> <definition>';
```

Comment:
Changed everything to `name` since the api side is working to resolve the `-` `_` name issue